spec/features/01_user_auth_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+feature 'user is able to' do
+
+    let!(:test_user) {User.create!(email: "test_user@mail.com", password: "password", username: "test_user_2", first_name: "test")}
+    
+    scenario 'click sign up' do
+        visit "/"
+
+        click_link "Sign Up"
+        checks = ["Username", "Password", "Email"]
+
+        checks.each do |check|
+            expect(page).to have_content(check)
+        end
```

Comment:
We don't need to ensure that those 3 fields are displayed on the page in their own test - the test below that interacts with those fields would be testing to make sure capybara can find form fields with those names